docker/centos6/Dockerfile

```diff
@@ -7,6 +7,8 @@ ENV INSTALL_DIR /opt/cloudera/parcels
 
 WORKDIR /BUILD
 
+RUN yum -y -e1 -d1 install https://download.postgresql.org/pub/repos/yum/reporpms/EL-6-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```

Comment:
Is it necessary to install the entire Postgresql Service or can we get away with just the client that you're installing below?

Reply:
Why are we choosing this repo?  Why not use Red Hat's SCL?

Reply:
I was having issues with other versions. pg_config is not being created. So I was forcing yum to install postgresql 9.4 and later updating the PATH variable. I was not using Red Hat's SCL repo because it does not have postgresql 9.4